tests/foreman/ui_airgun/test_package.py

```diff
@@ -0,0 +1,250 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Packages UI
+
+:Requirement: Packages
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+
+from fauxfactory import gen_string
+from nailgun import entities
+
+from robottelo import manifests
+from robottelo.api.utils import enable_rhrepo_and_fetchid
+from robottelo.constants import (
+    FAKE_0_YUM_REPO,
+    FAKE_3_YUM_REPO,
+    RPM_TO_UPLOAD,
+)
+
+from robottelo.decorators import fixture, tier2, upgrade
+from robottelo.helpers import get_data_file
+
+from robottelo.products import SatelliteToolsRepository
+
+
+@fixture(scope='module')
+def module_org():
+    return entities.Organization().create()
+
+
+@fixture(scope='module')
+def module_product(module_org):
+    return entities.Product(organization=module_org).create()
+
+
+@fixture(scope='module')
+def module_yum_repo(module_product):
+    yum_repo = entities.Repository(
+        name=gen_string('alpha'),
+        product=module_product,
+        content_type='yum',
+        url=FAKE_0_YUM_REPO,
+    ).create()
+    yum_repo.sync()
+    return yum_repo
+
+
+@fixture(scope='module')
+def module_yum_repo2(module_product):
+    yum_repo = entities.Repository(
+        name=gen_string('alpha'),
+        product=module_product,
+        content_type='yum',
+        url=FAKE_3_YUM_REPO,
+    ).create()
+    yum_repo.sync()
+    return yum_repo
+
+
+@fixture(scope='module')
+def module_rh_repo(module_org):
+    manifests.upload_manifest_locked(module_org.id, manifests.clone())
+    rhst = SatelliteToolsRepository(cdn=True)
+    repo_id = enable_rhrepo_and_fetchid(
+        basearch=rhst.data['arch'],
+        org_id=module_org.id,
+        product=rhst.data['product'],
+        repo=rhst.data['repository'],
+        reposet=rhst.data['repository-set'],
+        releasever=rhst.data['releasever'],
+    )
+    entities.Repository(id=repo_id).sync()
+    return entities.Repository(id=repo_id).read()
+
+
+@tier2
+def test_positive_search_in_repo(session, module_org, module_yum_repo):
+    """Create product with yum repository assigned to it. Search for
+    packages inside of it
+
+    :id: e182a89f-74e4-4b29-8152-1ea3bd014fd3
+
+    :expectedresults: Content search functionality works as intended and
+        expected packages are present inside of repository
+
+    :CaseLevel: Integration
+    """
+    with session:
+        session.organization.select(org_name=module_org.name)
+        assert session.package.search(
+            'name = bear', repository=module_yum_repo.name
+        )[0]['RPM'].startswith('bear')
```

Comment:
I was using `FAKE_0_CUSTOM_PACKAGE_NAME` and `FAKE_0_CUSTOM_PACKAGE` in my tests (`bear`/`bear-4.1-1.noarch`). Same with FAKE_1_CUSTOM_PACKAGE_NAME/FAKE_1_CUSTOM_PACKAGE. Maybe it's worth following the same approach if applicable? Not a blocker